[2016-06-26 21:26:05] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-26 21:26:18] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-26 21:26:31] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-26 21:26:42] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-26 21:29:29] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-26 21:31:55] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-26 21:35:29] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-26 21:35:46] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-26 21:40:57] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-26 21:42:14] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-26 21:44:41] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-26 21:45:12] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-26 21:48:02] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-26 22:09:46] local.ERROR: exception 'ErrorException' with message 'Use of undefined constant type - assumed 'type'' in C:\wamp64\www\System\storage\framework\views\638f8294b124b0efff4ed07b1322d1d8:197
Stack trace:
#0 C:\wamp64\www\System\storage\framework\views\638f8294b124b0efff4ed07b1322d1d8(197): Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(8, 'Use of undefine...', 'C:\\wamp64\\www\\S...', 197, Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(39): include('C:\\wamp64\\www\\S...')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(57): Illuminate\View\Engines\PhpEngine->evaluatePath('C:\\wamp64\\www\\S...', Array)
#3 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\View.php(142): Illuminate\View\Engines\CompilerEngine->get('C:\\wamp64\\www\\S...', Array)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\View.php(111): Illuminate\View\View->getContents()
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\View.php(80): Illuminate\View\View->renderContents()
#6 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Http\Response.php(44): Illuminate\View\View->render()
#7 C:\wamp64\www\System\vendor\symfony\http-foundation\Symfony\Component\HttpFoundation\Response.php(202): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#8 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1198): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#9 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(702): Illuminate\Routing\Router->prepareResponse(Object(Illuminate\Http\Request), Object(Illuminate\View\View))
#10 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#11 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(141): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#12 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#13 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(101): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(703): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(670): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#16 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(628): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#17 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(214): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#18 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#19 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(141): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#20 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\VerifyCsrfToken.php(43): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#21 C:\wamp64\www\System\app\Http\Middleware\VerifyCsrfToken.php(17): Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#22 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): App\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#23 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Middleware\ShareErrorsFromSession.php(55): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#24 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#25 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Session\Middleware\StartSession.php(61): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#26 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#27 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse.php(36): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#28 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#29 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\EncryptCookies.php(40): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#30 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#31 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(42): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#32 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#33 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#34 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(101): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#35 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(115): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#36 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(84): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#37 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#38 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#39 {main}

Next exception 'ErrorException' with message 'Use of undefined constant type - assumed 'type' (View: C:\wamp64\www\System\resources\views\facultyandstaff\admin\search-employee.blade.php)' in C:\wamp64\www\System\storage\framework\views\638f8294b124b0efff4ed07b1322d1d8:197
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(43): Illuminate\View\Engines\CompilerEngine->handleViewException(Object(ErrorException), 1)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(57): Illuminate\View\Engines\PhpEngine->evaluatePath('C:\\wamp64\\www\\S...', Array)
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\View.php(142): Illuminate\View\Engines\CompilerEngine->get('C:\\wamp64\\www\\S...', Array)
#3 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\View.php(111): Illuminate\View\View->getContents()
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\View.php(80): Illuminate\View\View->renderContents()
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Http\Response.php(44): Illuminate\View\View->render()
#6 C:\wamp64\www\System\vendor\symfony\http-foundation\Symfony\Component\HttpFoundation\Response.php(202): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#7 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1198): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#8 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(702): Illuminate\Routing\Router->prepareResponse(Object(Illuminate\Http\Request), Object(Illuminate\View\View))
#9 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#10 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(141): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#11 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#12 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(101): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#13 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(703): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#14 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(670): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#15 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(628): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#16 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(214): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#17 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#18 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(141): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#19 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\VerifyCsrfToken.php(43): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#20 C:\wamp64\www\System\app\Http\Middleware\VerifyCsrfToken.php(17): Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#21 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): App\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#22 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Middleware\ShareErrorsFromSession.php(55): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#23 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#24 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Session\Middleware\StartSession.php(61): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#25 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#26 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse.php(36): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#27 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#28 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\EncryptCookies.php(40): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#29 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#30 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(42): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#31 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#32 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#33 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(101): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#34 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(115): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#35 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(84): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#36 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#37 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#38 {main}  
[2016-06-26 22:10:31] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-26 22:10:45] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-26 22:13:01] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-26 22:13:06] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-26 22:13:08] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-26 22:13:11] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-26 22:13:38] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-26 22:14:52] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-26 22:15:07] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-26 22:15:47] local.ERROR: exception 'ErrorException' with message 'Undefined variable: type' in C:\wamp64\www\System\storage\framework\views\638f8294b124b0efff4ed07b1322d1d8:192
Stack trace:
#0 C:\wamp64\www\System\storage\framework\views\638f8294b124b0efff4ed07b1322d1d8(192): Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(8, 'Undefined varia...', 'C:\\wamp64\\www\\S...', 192, Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(39): include('C:\\wamp64\\www\\S...')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(57): Illuminate\View\Engines\PhpEngine->evaluatePath('C:\\wamp64\\www\\S...', Array)
#3 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\View.php(142): Illuminate\View\Engines\CompilerEngine->get('C:\\wamp64\\www\\S...', Array)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\View.php(111): Illuminate\View\View->getContents()
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\View.php(80): Illuminate\View\View->renderContents()
#6 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Http\Response.php(44): Illuminate\View\View->render()
#7 C:\wamp64\www\System\vendor\symfony\http-foundation\Symfony\Component\HttpFoundation\Response.php(202): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#8 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1198): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#9 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(702): Illuminate\Routing\Router->prepareResponse(Object(Illuminate\Http\Request), Object(Illuminate\View\View))
#10 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#11 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(141): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#12 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#13 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(101): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(703): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(670): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#16 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(628): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#17 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(214): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#18 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#19 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(141): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#20 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\VerifyCsrfToken.php(43): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#21 C:\wamp64\www\System\app\Http\Middleware\VerifyCsrfToken.php(17): Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#22 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): App\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#23 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Middleware\ShareErrorsFromSession.php(55): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#24 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#25 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Session\Middleware\StartSession.php(61): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#26 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#27 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse.php(36): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#28 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#29 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\EncryptCookies.php(40): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#30 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#31 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(42): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#32 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#33 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#34 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(101): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#35 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(115): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#36 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(84): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#37 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#38 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#39 {main}

Next exception 'ErrorException' with message 'Undefined variable: type (View: C:\wamp64\www\System\resources\views\facultyandstaff\admin\search-employee.blade.php)' in C:\wamp64\www\System\storage\framework\views\638f8294b124b0efff4ed07b1322d1d8:192
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(43): Illuminate\View\Engines\CompilerEngine->handleViewException(Object(ErrorException), 1)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(57): Illuminate\View\Engines\PhpEngine->evaluatePath('C:\\wamp64\\www\\S...', Array)
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\View.php(142): Illuminate\View\Engines\CompilerEngine->get('C:\\wamp64\\www\\S...', Array)
#3 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\View.php(111): Illuminate\View\View->getContents()
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\View.php(80): Illuminate\View\View->renderContents()
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Http\Response.php(44): Illuminate\View\View->render()
#6 C:\wamp64\www\System\vendor\symfony\http-foundation\Symfony\Component\HttpFoundation\Response.php(202): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#7 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1198): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#8 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(702): Illuminate\Routing\Router->prepareResponse(Object(Illuminate\Http\Request), Object(Illuminate\View\View))
#9 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#10 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(141): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#11 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#12 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(101): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#13 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(703): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#14 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(670): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#15 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(628): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#16 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(214): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#17 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#18 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(141): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#19 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\VerifyCsrfToken.php(43): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#20 C:\wamp64\www\System\app\Http\Middleware\VerifyCsrfToken.php(17): Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#21 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): App\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#22 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Middleware\ShareErrorsFromSession.php(55): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#23 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#24 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Session\Middleware\StartSession.php(61): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#25 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#26 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse.php(36): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#27 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#28 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\EncryptCookies.php(40): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#29 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#30 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(42): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#31 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#32 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#33 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(101): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#34 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(115): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#35 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(84): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#36 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#37 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#38 {main}  
[2016-06-26 22:15:48] local.ERROR: exception 'ErrorException' with message 'Undefined variable: type' in C:\wamp64\www\System\storage\framework\views\638f8294b124b0efff4ed07b1322d1d8:192
Stack trace:
#0 C:\wamp64\www\System\storage\framework\views\638f8294b124b0efff4ed07b1322d1d8(192): Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(8, 'Undefined varia...', 'C:\\wamp64\\www\\S...', 192, Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(39): include('C:\\wamp64\\www\\S...')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(57): Illuminate\View\Engines\PhpEngine->evaluatePath('C:\\wamp64\\www\\S...', Array)
#3 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\View.php(142): Illuminate\View\Engines\CompilerEngine->get('C:\\wamp64\\www\\S...', Array)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\View.php(111): Illuminate\View\View->getContents()
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\View.php(80): Illuminate\View\View->renderContents()
#6 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Http\Response.php(44): Illuminate\View\View->render()
#7 C:\wamp64\www\System\vendor\symfony\http-foundation\Symfony\Component\HttpFoundation\Response.php(202): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#8 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1198): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#9 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(702): Illuminate\Routing\Router->prepareResponse(Object(Illuminate\Http\Request), Object(Illuminate\View\View))
#10 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#11 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(141): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#12 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#13 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(101): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(703): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(670): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#16 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(628): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#17 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(214): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#18 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#19 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(141): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#20 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\VerifyCsrfToken.php(43): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#21 C:\wamp64\www\System\app\Http\Middleware\VerifyCsrfToken.php(17): Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#22 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): App\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#23 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Middleware\ShareErrorsFromSession.php(55): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#24 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#25 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Session\Middleware\StartSession.php(61): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#26 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#27 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse.php(36): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#28 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#29 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\EncryptCookies.php(40): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#30 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#31 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(42): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#32 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#33 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#34 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(101): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#35 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(115): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#36 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(84): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#37 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#38 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#39 {main}

Next exception 'ErrorException' with message 'Undefined variable: type (View: C:\wamp64\www\System\resources\views\facultyandstaff\admin\search-employee.blade.php)' in C:\wamp64\www\System\storage\framework\views\638f8294b124b0efff4ed07b1322d1d8:192
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(43): Illuminate\View\Engines\CompilerEngine->handleViewException(Object(ErrorException), 1)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(57): Illuminate\View\Engines\PhpEngine->evaluatePath('C:\\wamp64\\www\\S...', Array)
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\View.php(142): Illuminate\View\Engines\CompilerEngine->get('C:\\wamp64\\www\\S...', Array)
#3 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\View.php(111): Illuminate\View\View->getContents()
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\View.php(80): Illuminate\View\View->renderContents()
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Http\Response.php(44): Illuminate\View\View->render()
#6 C:\wamp64\www\System\vendor\symfony\http-foundation\Symfony\Component\HttpFoundation\Response.php(202): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#7 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1198): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#8 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(702): Illuminate\Routing\Router->prepareResponse(Object(Illuminate\Http\Request), Object(Illuminate\View\View))
#9 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#10 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(141): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#11 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#12 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(101): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#13 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(703): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#14 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(670): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#15 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(628): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#16 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(214): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#17 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#18 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(141): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#19 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\VerifyCsrfToken.php(43): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#20 C:\wamp64\www\System\app\Http\Middleware\VerifyCsrfToken.php(17): Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#21 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): App\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#22 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Middleware\ShareErrorsFromSession.php(55): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#23 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#24 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Session\Middleware\StartSession.php(61): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#25 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#26 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse.php(36): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#27 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#28 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\EncryptCookies.php(40): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#29 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#30 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(42): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#31 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#32 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#33 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(101): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#34 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(115): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#35 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(84): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#36 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#37 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#38 {main}  
[2016-06-26 22:15:49] local.ERROR: exception 'ErrorException' with message 'Undefined variable: type' in C:\wamp64\www\System\storage\framework\views\638f8294b124b0efff4ed07b1322d1d8:192
Stack trace:
#0 C:\wamp64\www\System\storage\framework\views\638f8294b124b0efff4ed07b1322d1d8(192): Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(8, 'Undefined varia...', 'C:\\wamp64\\www\\S...', 192, Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(39): include('C:\\wamp64\\www\\S...')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(57): Illuminate\View\Engines\PhpEngine->evaluatePath('C:\\wamp64\\www\\S...', Array)
#3 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\View.php(142): Illuminate\View\Engines\CompilerEngine->get('C:\\wamp64\\www\\S...', Array)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\View.php(111): Illuminate\View\View->getContents()
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\View.php(80): Illuminate\View\View->renderContents()
#6 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Http\Response.php(44): Illuminate\View\View->render()
#7 C:\wamp64\www\System\vendor\symfony\http-foundation\Symfony\Component\HttpFoundation\Response.php(202): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#8 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1198): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#9 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(702): Illuminate\Routing\Router->prepareResponse(Object(Illuminate\Http\Request), Object(Illuminate\View\View))
#10 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#11 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(141): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#12 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#13 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(101): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(703): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(670): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#16 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(628): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#17 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(214): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#18 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#19 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(141): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#20 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\VerifyCsrfToken.php(43): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#21 C:\wamp64\www\System\app\Http\Middleware\VerifyCsrfToken.php(17): Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#22 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): App\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#23 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Middleware\ShareErrorsFromSession.php(55): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#24 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#25 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Session\Middleware\StartSession.php(61): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#26 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#27 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse.php(36): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#28 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#29 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\EncryptCookies.php(40): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#30 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#31 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(42): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#32 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#33 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#34 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(101): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#35 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(115): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#36 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(84): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#37 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#38 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#39 {main}

Next exception 'ErrorException' with message 'Undefined variable: type (View: C:\wamp64\www\System\resources\views\facultyandstaff\admin\search-employee.blade.php)' in C:\wamp64\www\System\storage\framework\views\638f8294b124b0efff4ed07b1322d1d8:192
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(43): Illuminate\View\Engines\CompilerEngine->handleViewException(Object(ErrorException), 1)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(57): Illuminate\View\Engines\PhpEngine->evaluatePath('C:\\wamp64\\www\\S...', Array)
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\View.php(142): Illuminate\View\Engines\CompilerEngine->get('C:\\wamp64\\www\\S...', Array)
#3 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\View.php(111): Illuminate\View\View->getContents()
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\View.php(80): Illuminate\View\View->renderContents()
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Http\Response.php(44): Illuminate\View\View->render()
#6 C:\wamp64\www\System\vendor\symfony\http-foundation\Symfony\Component\HttpFoundation\Response.php(202): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#7 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1198): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#8 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(702): Illuminate\Routing\Router->prepareResponse(Object(Illuminate\Http\Request), Object(Illuminate\View\View))
#9 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#10 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(141): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#11 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#12 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(101): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#13 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(703): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#14 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(670): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#15 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(628): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#16 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(214): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#17 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#18 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(141): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#19 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\VerifyCsrfToken.php(43): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#20 C:\wamp64\www\System\app\Http\Middleware\VerifyCsrfToken.php(17): Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#21 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): App\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#22 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Middleware\ShareErrorsFromSession.php(55): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#23 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#24 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Session\Middleware\StartSession.php(61): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#25 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#26 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse.php(36): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#27 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#28 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\EncryptCookies.php(40): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#29 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#30 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(42): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#31 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#32 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#33 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(101): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#34 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(115): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#35 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(84): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#36 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#37 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#38 {main}  
[2016-06-26 22:15:57] local.ERROR: exception 'ErrorException' with message 'Undefined variable: type' in C:\wamp64\www\System\storage\framework\views\638f8294b124b0efff4ed07b1322d1d8:192
Stack trace:
#0 C:\wamp64\www\System\storage\framework\views\638f8294b124b0efff4ed07b1322d1d8(192): Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(8, 'Undefined varia...', 'C:\\wamp64\\www\\S...', 192, Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(39): include('C:\\wamp64\\www\\S...')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(57): Illuminate\View\Engines\PhpEngine->evaluatePath('C:\\wamp64\\www\\S...', Array)
#3 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\View.php(142): Illuminate\View\Engines\CompilerEngine->get('C:\\wamp64\\www\\S...', Array)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\View.php(111): Illuminate\View\View->getContents()
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\View.php(80): Illuminate\View\View->renderContents()
#6 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Http\Response.php(44): Illuminate\View\View->render()
#7 C:\wamp64\www\System\vendor\symfony\http-foundation\Symfony\Component\HttpFoundation\Response.php(202): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#8 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1198): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#9 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(702): Illuminate\Routing\Router->prepareResponse(Object(Illuminate\Http\Request), Object(Illuminate\View\View))
#10 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#11 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(141): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#12 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#13 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(101): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(703): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(670): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#16 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(628): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#17 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(214): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#18 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#19 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(141): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#20 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\VerifyCsrfToken.php(43): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#21 C:\wamp64\www\System\app\Http\Middleware\VerifyCsrfToken.php(17): Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#22 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): App\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#23 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Middleware\ShareErrorsFromSession.php(55): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#24 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#25 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Session\Middleware\StartSession.php(61): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#26 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#27 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse.php(36): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#28 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#29 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\EncryptCookies.php(40): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#30 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#31 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(42): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#32 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#33 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#34 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(101): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#35 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(115): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#36 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(84): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#37 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#38 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#39 {main}

Next exception 'ErrorException' with message 'Undefined variable: type (View: C:\wamp64\www\System\resources\views\facultyandstaff\admin\search-employee.blade.php)' in C:\wamp64\www\System\storage\framework\views\638f8294b124b0efff4ed07b1322d1d8:192
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(43): Illuminate\View\Engines\CompilerEngine->handleViewException(Object(ErrorException), 1)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(57): Illuminate\View\Engines\PhpEngine->evaluatePath('C:\\wamp64\\www\\S...', Array)
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\View.php(142): Illuminate\View\Engines\CompilerEngine->get('C:\\wamp64\\www\\S...', Array)
#3 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\View.php(111): Illuminate\View\View->getContents()
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\View.php(80): Illuminate\View\View->renderContents()
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Http\Response.php(44): Illuminate\View\View->render()
#6 C:\wamp64\www\System\vendor\symfony\http-foundation\Symfony\Component\HttpFoundation\Response.php(202): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#7 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1198): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#8 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(702): Illuminate\Routing\Router->prepareResponse(Object(Illuminate\Http\Request), Object(Illuminate\View\View))
#9 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#10 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(141): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#11 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#12 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(101): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#13 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(703): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#14 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(670): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#15 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\Router.php(628): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#16 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(214): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#17 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#18 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(141): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#19 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\VerifyCsrfToken.php(43): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#20 C:\wamp64\www\System\app\Http\Middleware\VerifyCsrfToken.php(17): Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#21 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): App\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#22 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Middleware\ShareErrorsFromSession.php(55): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#23 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#24 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Session\Middleware\StartSession.php(61): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#25 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#26 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse.php(36): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#27 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#28 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\EncryptCookies.php(40): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#29 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#30 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(42): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#31 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#32 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#33 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(101): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#34 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(115): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#35 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(84): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#36 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#37 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#38 {main}  
[2016-06-26 22:16:07] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-26 22:16:50] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-26 22:16:52] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-26 22:16:57] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-26 22:17:02] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-26 22:18:23] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-26 22:18:54] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-26 22:25:56] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-26 22:27:59] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-26 22:28:01] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-26 22:32:37] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-26 22:32:51] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-26 22:33:15] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-26 22:33:18] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-26 22:33:21] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-26 22:33:27] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-26 22:33:31] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-26 22:33:33] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-26 22:33:35] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-26 22:33:38] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-26 22:33:48] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-26 22:33:49] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-26 22:33:51] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
[2016-06-26 22:33:59] local.ERROR: exception 'InvalidArgumentException' with message 'View [errors.404] not found.' in C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:140
Stack trace:
#0 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(77): Illuminate\View\FileViewFinder->findInPaths('errors.404', Array)
#1 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\View\Factory.php(145): Illuminate\View\FileViewFinder->find('errors.404')
#2 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(68): Illuminate\View\Factory->make('errors.404', Array)
#3 C:\wamp64\www\System\app\Exceptions\Handler.php(42): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404)
#4 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(248): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#5 C:\wamp64\www\System\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(90): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#6 C:\wamp64\www\System\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#7 C:\wamp64\www\System\server.php(21): require_once('C:\\wamp64\\www\\S...')
#8 {main}  
